test/test_alert_types.cpp

```diff
@@ -65,9 +65,16 @@ TORRENT_TEST(alerts_types)
 #else
 	++count_alert_types;
 #endif
+
+#ifndef TORRENT_NO_DEPRECATE
+#define PROGRESS_NOTIFICATION alert::progress_notification |
+#else
+#define PROGRESS_NOTIFICATION
```

Comment:
maybe you should add `#undef PROGRESS_NOTIFICATION` after the tests (like `#undef TEST_ALERT_TYPE`)